app/models/transformation_mapping/cloud_best_fit.rb

```diff
@@ -0,0 +1,20 @@
+class TransformationMapping
+  class CloudBestFit
+    attr_reader :source_vm, :destination_manager
+    def initialize(source_vm, destination_manager)
+      @source_vm = source_vm
+      @destination_manager = destination_manager
+    end
+
+    def available_fit_flavors
+      @available_fit_flavors ||= destination_manager.flavors.where(
+        :cpus   => source_vm.cpu_total_cores..Float::INFINITY,
```

Comment:
@agrare Are there any other attributes that we need to match up?

Reply:
@agrare Do we need to worry about root disk or any of those attributes?

Reply:
Ohh thought you meant specifically about CPUs.  I don't know how/if the flavor's root_disk_size is used by the virt-v2v migration tool.  @aufi @fdupont-redhat do you know?

Reply:
ping @aufi @mansam @fdupont-redhat do you guy know what is required re: target flavor

Reply:
I don't know about virt-v2v tool, but from OpenStack point of view, I believe we should care about _root_disk_size_ in similar way as RAM. So, yes, please add _root_disk_size_ too.

Reply:
@bthurber Do you know how we determine which disk is the "root disk" for v2v?

Reply:
@bdunne this is determined by virt-v2v.  I'll tee up a conversation with rjones.

Reply:
The complete list of roots is determined by [libguestfs inspection](http://libguestfs.org/guestfs.3.html#inspection).  Most guests will have only one, but you can influence which root is chosen (eg for dual boot guests) using the [`--root` option](http://libguestfs.org/virt-v2v.1.html#options).

Reply:
Having said that, reading back, I'm not exactly sure what you're asking about.

Reply:
BTW I've never been able to get github to send me emails about new messages, so if you write something here most likely I will never see it.  Best to email me with questions, or talk about this on a non-proprietary service such as the v2v-devel (private) or libguestfs (public) mailing lists.

Reply:
Sounds like we won't know which disk is the root disk, so we can:
- merge as is
or
- get the size of the first disk if there is only one

Reply:
Unless we have this information from fleecing the VM

Reply:
@agrare TBH, I don't know. We should test it.
@rwjones do you have an environment to test it ?

Reply:
@bdunne I say we merge this as-is with cpu/memory and come back and revisit after we get clarity on the disk size issue.